src/components/Button/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { TouchableWithoutFeedback } from 'react-native';
+import Typography from '../Typography';
+import { Button } from './styles';
+
+const ButtonMain = ({ handlePress, color, styleTypho, string }) => {
+  return (
+    <TouchableWithoutFeedback onPress={handlePress}>
+      <Button color={color}>
+        <Typography style={styleTypho}>{string}</Typography>
+      </Button>
+    </TouchableWithoutFeedback>
+  );
+};
```

Comment:
Aqui no lugar de string tu pode passar o children que fica um pouco mais REACT Style (fazer alterações respectivas nas chamadas)
```suggestion
const ButtonMain = ({ handlePress, color, styleTypho, children }) => {
  return (
    <TouchableWithoutFeedback onPress={handlePress}>
      <Button color={color}>
        <Typography style={styleTypho}>{children}</Typography>
      </Button>
    </TouchableWithoutFeedback>
  );
};
```

Reply:
De acordo!